bundle/manifests/cryostat-operator.clusterserviceversion.yaml

```diff
@@ -180,6 +180,24 @@ spec:
       - description: Options to customize the storage for Flight Recordings and Templates
         displayName: Storage Options
         path: storageOptions
+      - description: Configuration for an EmptyDir to be created by the operator instead of a PVC. If an EmptyDir configuration is enabled, any PVC configurations will be ignored.
+        displayName: Empty Dir
+        path: storageOptions.emptyDir
+      - description: When enabled, Cryostat will use EmptyDir volumes instead of a Persistent Volume Claim.
+        displayName: Enabled
+        path: storageOptions.emptyDir.enabled
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+      - description: Unless specified, the emptyDir volume will be mounted on the same storage medium backing the node. Setting this field to "Memory" will mount the emptyDir on a tmpfs (RAM-backed filesystem).
+        displayName: Medium
+        path: storageOptions.emptyDir.medium
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:spec.storageOptions.emptyDir.enabled:true
+      - description: The maximum memory limit for the emptyDir. Default is unbounded.
+        displayName: Size Limit
+        path: storageOptions.emptyDir.sizeLimit
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:spec.storageOptions.emptyDir.enabled:true
```

Comment:
For the `fieldDependency` descriptors, the `spec` property is already implied, so these should begin with `storageOptions`.

There's unfortunately another problem that prevents `sizeLimit` from appearing in the OpenShift Console form view: https://github.com/openshift/console/issues/8298. `resource.Quantity` is treated as an int-or-string type by controller-tools, which is used by the Operator SDK: https://github.com/kubernetes-sigs/controller-tools/blob/97ee66cc83b03bd632cf8da35496812ca091ae4e/pkg/crd/known_types.go#L61-L70

We could work around this by changing `sizeLimit` to a `*string` and adding the validation manually with:
`// +kubebuilder:validation:Pattern=^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$` and then parsing the string into a Quantity with `resource.ParseQuantity`. What do you think?

Reply:
That's unfortunate about the `sizeLimit` form field. I've changed the field to a string and added the validation you suggested.